operators/src/mock/mock_raster_source.rs

```diff
@@ -18,14 +21,28 @@ where
     T: Pixel,
 {
     pub data: Vec<RasterTile2D<T>>,
+    pub no_data_value: Option<T>,
+    pub tiling_specification: TilingSpecification,
```

Comment:
Why don't we just use this in (from) the initialize process and fail then?

Reply:
we get the tiling_spec from the operator when it is initialized